### ISSUE TITLE
Consensus Clustering-Based Undersampling Approach to Imbalanced Learning

### ISSUE THREAD

**Issue by ShifaTang** [AbdulKhadeer2003/Consensus-Clustering-based-undersampling-approach#1]
Dude, is this the replication of the code for the paper 'Consensus Clustering-Based Undersampling Approach to Imbalanced Learning

**Comment by AbdulKhadeer2003** [comment]
Yes Dude, it replication of various research papers related to Consensus Clustering-Based Undersampling Approach to Imbalanced Learning.
